### DISCUSSION TITLE
What's the best way to add the Media Renditions API to `mux-video-react`?

### DISCUSSION THREAD

**Discussion by gogocharli** [muxinc/elements#910] [Q&A] [answered]
Unlike `mux-video`, `mux-video-react` doesn't seem to support the media renditions API by default. We would like to be able to display a quality selector with media-chrome, but simply using the mixin from `media-tracks` doesn't seem to work. Is there a better way to approach this issue?

**Comment by luwes** [comment, marked as answer]
I think the easiest workaround for now is to use `<mux-video>` in React if you're okay having a custom element in your markup.

We've done the same in https://player.style!
https://github.com/muxinc/player.style/blob/main/site/app/_components/Video.tsx
